src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/Model/PPASError.swift

```diff
@@ -12,9 +12,16 @@ enum PPASError: Error {
 	case jsonError
 	case serverError(PPAServerErrorCode)
 	case serverFailure(Error)
+	case ppacError
+	case appResetError
+	case onboardingError
+	case submission23hoursError
+	case probibilityError
+	case userConsentError
 }
 
 extension PPASError: Equatable {
+	// swiftlint:disable cyclomatic_complexity
 	static func == (lhs: PPASError, rhs: PPASError) -> Bool {
 		switch(lhs, rhs) {
```

Comment:
Maybe this helps – at least it does in my project ;)
`lhs.localizedDescription == rhs.localizedDescription`

Reply:
Enums default implicitly to `Int` and this would generate something like 
> "The operation couldn’t be completed. (PPASError error 42.)"

Reply:
Nice approach. Let's try it 👍